### ISSUE TITLE
Identify VIEW, MAIN w/ tabs bottleneck

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/fenix#16328]
We know the MAIN bottleneck is the main thread: we should see what it is for our other start up paths.

**Comment by MarcLeclair** [comment]
Tested with VIEW with tabs bottleneck. It seems as if when we created a bottleneck: 

- on the CPU the G5 goes from  2.30332 seconds  to 3.52312 seconds 

- By writing to a large file, it goes from 2.30332 seconds to  3.24190 seconds

- With both CPU and writing to a large file the G5 goes from 2.30332 seconds to 574960 seconds

However, this needs to be tested with the "new way" to open Fenix which is to the latest tab used. Right now, FNPRMS takes in the "MAIN" intent which starts without a URL and expects a `Fully Drawn` log. However, this doesn't appear with the path where "MAIN" has a tab already opened.  This means that running "MAIN" with a opened tab and using `times.py` using the "VIEW" option, will not give any averages.
